app/schemas/user.py

```
@@ -2,13 +2,34 @@
+    code = v.removeprefix("+")
+    if not code.isdigit() or not (1 <= len(code) <= 3):
+        raise ValueError("Phone region must be a 1, 2, or 3-digit number.")
+    return v
+
+
+def validate_phone_number(v: str) -> str:
+    if not isinstance(v, str):
+        return v
+
+    if not v.isdigit() or not (8 <= len(v) <= 9):
+        raise ValueError("Phone number must be an 8 or 9-digit number.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)